gapis/api/vulkan/api/descriptor.api

```diff
@@ -729,26 +729,15 @@ emptyBufferOffsets {
 }
 
 sub void updateDescriptorBufferBindingOffsets(ref!vkCmdBindDescriptorSetsArgs args) {
-  _ = PipelineLayouts[args.Layout]
```

Comment:
This line was added to fix missing dependencies between `vkCmdBindDescriptorSets` and `vkCreatePipelineLayout`. Is there any reason to think it is no longer necessary?

Reply:
Because it is also in trackDescriptorBufferBindingOffsets(), which this calls.